app/components/wds-avatar-stack.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+import {computed} from '@ember/object';
+
+
+export default Component.extend({
+	classNames: ['wds-avatar-stack'],
+	overflow: computed('avatars', function() {
+		const count = this.get('avatars').length;
+
+		return count > 5 ? this.get('avatars').length - 5 : 0;
```

Comment:
this 5 could be a property, so it can be overriden if needed

Reply:
count - 5?

Reply:
@hakubo I don't think we can allow for more elements than 5 unless we'll figure out how to handle it better in scss: https://github.com/Wikia/design-system/blob/3e6f170b7379a523f3ce6c5def7db03bab063405/dist/scss/wds-components/_avatar-stack.scss#L10

Reply:
I would extract it to the property and leave a comment that scss file should be updated together with this value.